src/customJs/socialShareTool/SocialShareTool.js

```diff
@@ -19,51 +19,71 @@ export const SocialShareTool = (props) => {
                 valign="middle"
               >
                 {values.social_share_network_facebook && (
-                  <a href="#" target="_blank">
+                  <a
+                    socialshare="4"
+                    href="[[[$share_vo_facebook]]]"
```

Comment:
Give me some minutes before merging.

Based [on this](https://github.com/MakingSense/Doppler/blob/02c33808dbcb72efa46d54c885d5003dd56e4d3e/Doppler.Presentation.MVC/App.Release.Config#L330-L335), embedded tags have only two `[`.

We could use the same standard.

Reply:
```suggestion
                    href="[[shareInFacebook]]]"
```

CC: @Cromeror

Reply:
I think it could be `shareLinkFacebook` or `shareInFacebook`

Reply:
I agree, in my opinion `Link` is wrong because these are URLs not links.

But, current embedded tags follow the other format: `[[remotionLink]]`, `[[linkRemocion]]`, `[[viewOnlineLink]]`, `[[linkVerOnline]]`, `[[forwardLink]]`, `[[linkReenviar]]`, `[[updateProfileLink]]`, `[[linkActualizarPerfil]]`, see 
https://github.com/MakingSense/Doppler/blob/02c33808dbcb72efa46d54c885d5003dd56e4d3e/Doppler.Presentation.MVC/App.Release.Config#L332-L335

Reply:
Ok, agree!